templates/understanding-urbit/list.html

```diff
@@ -8,65 +8,39 @@
   <a href="/understanding-urbit/">
     Understanding Urbit
   </a>
-  {% include "partials/navigation-search.html" %}
```

Comment:
Did you intend to remove the Search from the sidebar on UU's intro, as well? And the sidebar is on the other pages — if you want to remove it from all of them, you'll have to do that again in `/templates/understanding-urbit/post.html`